### PR TITLE
🎨 raise phpstan level

### PR DESCRIPTION
**Pull request by garak**
This will likely decrease coverage a bit, since it's really hard to cover edge case (but such cases are caught by phpstan anyway)

### CODE REVIEW

File: tests/PgnTest.php [hunk 1]
```diff
@@ -148,6 +144,11 @@ public function testValidatePgn(): void
         $this->assertTrue($parsed);
 
         $parsed = Validation::validatePgn('1.e4 e5 2.Nf3', ['verbose' => true]);
+        if (!\is_array($parsed)) {
+            $this->markTestSkipped();
```
Comment (akondas) [inline tests/PgnTest.php:148]: Why skipped, shouldn't be failed?

Comment (garak) [reply tests/PgnTest.php:148]: Well, I guess it's the same. It's a case limit and actually it should never be something different from an array

Comment (akondas) [reply tests/PgnTest.php:148]: then it is better to throw an exception, in some cases skipped can be treated as ok :wink:

**Comment by akondas** [comment]
Thanks :+1:
